README.md

```diff
@@ -4,14 +4,28 @@ This app serves Reader Revenue messages (epics and banners) to theguardian.com.
 
 See [architecture](docs/architecture.md) for details.
 
+## Setup
+
+To set up, first run 
+`nvm use`
+`yarn`
+
+Then you can follow the instructions below to run the server locally.
+
 ## Development
 
-### Run the dev server
+You may want to run in conjunction with DCR, or you can use storybook if you just want to develop quickly.
 
-```
-$ yarn dev
-$ PORT=8080 yarn dev # override the port to listen on
-```
+### Run the dev server for use with DCR
+
+`NODE_OPTIONS="--max-old-space-size=80000" AWS_PROFILE=membership PORT=8082 yarn dev`
+
+Then you can access it on
+http://localhost:8080
```

Comment:
I think we just want: `http://localhost:8082` here. Also (and I'm happy to add this if you'd like!) we need to point our local DCR instance at the local SDC instance. I'd normally do that by updating this line here:  https://github.com/guardian/dotcom-rendering/blob/main/src/web/components/SlotBodyEnd/ReaderRevenueEpic.tsx#L178

to something like

```ts
const response = await getBodyEnd(
		contributionsPayload,
		`http://localhost:8082/epic${queryString}`,
	);
```

Reply:
sounds good, feel free to add some commits/merge this/etc!  or I'll get to it next week probably